packages/generator-prismic-nextjs/generators/slicemachine/templates/customtypes/page/index.json

```diff
@@ -2,6 +2,8 @@
   "id": "page",
   "label": "Page",
   "repeatable": true,
+  "status": true,
+  "description": "",
```

Comment:
no need for description on the custom type level, it was a confusion with shared slices.

Reply:
Cool I'll get that removed :)